README.md

```diff
@@ -1,4 +1,8 @@
-# coyim - a safe and secure chat client
+<p align="center">
```

Comment:
I think for a markdown file, I'm not super comfortable with adding the logo here.